### ISSUE TITLE
It is not clear how to use this

### ISSUE THREAD

**Issue by Elendil211** [city96/ComfyUI-GGUF#71]
I tried using this, and I failed on knowing what to download and how to configure the program.

The readme only mentions how to use the unet model, but doesn't mention how to use CLIP and VAE.

I tried guessing:

* VAE: LoadVAE with either 
** https://huggingface.co/black-forest-labs/FLUX.1-schnell/blob/main/ae.safetensors or
** https://huggingface.co/black-forest-labs/FLUX.1-schnell/blob/main/vae/diffusion_pytorch_model.safetensors
* CLIP, either:
** CLIPLoader with https://huggingface.co/city96/t5-v1_1-xxl-encoder-gguf/tree/main
** or DualCLIPLoader with a combination of https://huggingface.co/black-forest-labs/FLUX.1-schnell/blob/main/text_encoder/model.safetensors and https://huggingface.co/city96/t5-v1_1-xxl-encoder-gguf/tree/main

But none of this works, and the CLIP Text Encode just never finishes without any indication of what is wrong.

**Comment by city96** [comment]
Yeah, the readme really needs to be rewritten. You're supposed to use the DualCLIPLoader, which needs both the T5 and the clip-l model with the mode set to "flux". The clip-l and default T5 models are [here](https://huggingface.co/comfyanonymous/flux_text_encoders/tree/main). Even with the gguf one you still need the regular clip_l.safetensors one.

**Comment by Elendil211** [comment]
And what do I use for VAE?

**Comment by city96** [comment]
The one you linked is correct, the quantization stuff doesn't change anything about the VAE. (There also an example page [here](https://comfyanonymous.github.io/ComfyUI_examples/flux/) that links to all the default models).

**Comment by Elendil211** [comment]
Which one, though? I linked two options:

* https://huggingface.co/black-forest-labs/FLUX.1-schnell/blob/main/ae.safetensors or
* https://huggingface.co/black-forest-labs/FLUX.1-schnell/blob/main/vae/diffusion_pytorch_model.safetensors

**Comment by al-swaiti** [comment]
this workflow could help  u https://civitai.com/models/652981/gguf-workflow-simple

**Comment by B0rner** [comment]
@city96 
> Yeah, the readme really needs to be rewritten.

Could it be possible to place a workflow file in that repository with a setup, that is max. simple as possible?

The linked workflow from @al-swaiti for example needs an gemini plugin. I'm new to comfyui and I would only like to make that running - without using any external ressources, like on gemini and I don't know, how to remove the ressource from the linked workflow. I found an other workflow on civitai which is much mor complicated, that need many custom_node. I have no idea, what they all doing and if they are only needed for using gguf.

**Comment by city96** [comment]
@B0rner Don't have much free time lately but if all you want is a super basic workflow then this should do.

Btw, all you need to do to use this node pack is replacing the unet loader node (and optionally the dual clip loader node) with the gguf variants. There's zero other dependencies or custom nodes required to make it work, and you could even use the default comfy example workflow as a base for this.

[base_flux_gguf_test.json](https://github.com/user-attachments/files/17062314/base_flux_gguf_test.json)

![base_flux_gguf_test](https://github.com/user-attachments/assets/bca79ebb-6496-4f92-8fa3-30143bf2e7af)

(If you want to use cfg >1 or any of the fancy cfg stuff just add a second CLIP text encode node instead of connecting both positive/negative to the same one. At cfg=1 the negative prompt does nothing so this doesn't matter in this case.)

**Comment by B0rner** [comment]
@city96 

Thank you. That was very helpful. I had some issues, from where to downlod what file and where to place it (becaus I'm new to comfy), but finally everythink is working fine. Using Flux with GGUF is realy great. Generatig images on my small laptop are much faster than using Flux-16bit.

**Comment by trinhtuanvubk** [comment]
Hi @city96 . What should I do if I want to leave nagative empty but set cfg > 1. I have run your above workflow in python inference successfully. Could you give me a new pic of workflow or some details like node name, class name, node connection ,... to help me convert to code easily. Thank you

**Comment by city96** [comment]
@trinhtuanvubk Either of these should work in that case. You can modify the above workflow accordingly.

![image](https://github.com/user-attachments/assets/66a4b51a-1432-4abc-b3c4-a07a4d7e3998)

![image](https://github.com/user-attachments/assets/8c7bd320-df0e-4e97-a7a1-9ae71c11325c)
